src/invidious/helpers/serialized_yt_data.cr

```diff
@@ -0,0 +1,258 @@
+struct SearchVideo
```

Comment:
Adding more "miscellanous" files to the `helpers` folder seems a bad idea. What do you think about having a dedicated `data_struct` folder, where we will put the large and common data structures, one per file?

The idea would be to be able, in the end, to `require` a specific structure, without having to require the rest of the project every time, and probably improve build/testing time.

Reply:
Huh that's a good idea, Invidious structs would be far more organized that way. Though it really shouldn't affect build times too much. I don't understand the nitty gritty of the Crystal compiler but:

If `fileA` requires `fileC` and  `fileB` requires `fileA` then the contents of `fileC` is going to be available in the global namespace of `fileB` including whatever else it imports. In Invidious, since all of the imports are done in `invidious.cr` everything is already available from the get-go and any additional imports after that are redundant and likely removed by Crystal to my knowledge. 
 
I'll go ahead and make the change to see how it affects buildtimes though.